airflow/operators/python.py

```diff
@@ -517,6 +521,16 @@ def __deepcopy__(self, memo):
         memo[id(self.pickling_library)] = self.pickling_library
         return super().__deepcopy__(memo)
 
+    def __getstate__(self):
+        state = super().__getstate__()
+        del state['pickling_library']
+
+        return state
+
+    def __setstate__(self, state):
+        super().__setstate__(state)
+        self.pickling_library = dill if self.use_dill else pickle
```

Comment:
Hmm this attribute should probably have been an `@property` to begin with. In which case we don’t even need to worry about pickling it at all.